custom_components/plugwise/climate.py

```diff
@@ -250,11 +250,11 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
         """Set new target temperature."""
         data: dict[str, Any] = {}
         if ATTR_TEMPERATURE in kwargs:
-            data[SETPOINT] = kwargs.get(ATTR_TEMPERATURE)
```

Comment:
setpoint is a pretty industry standard ... or is HA moving to target temp?

Reply:
I've used the HA climate constant "names" but kept our strings.
I'm thinking, lining up the constant-names is less confusing on HA Core level.

Reply:
If it's HA than lets go that route